src/crepe/facade/SDLFontContext.cpp

```diff
@@ -0,0 +1,53 @@
+#include <stdexcept>
+
+#include "SDLFontContext.h"
+
+using namespace crepe;
+using namespace std;
+
+SDLFontContext::SDLFontContext() {
+	if (!FcInit()) {
+		throw runtime_error("Failed to initialize Fontconfig.");
+	}
+}
+
+SDLFontContext::~SDLFontContext() { FcFini(); }
+
+unique_ptr<Asset> SDLFontContext::get_font_asset(const string font_family) {
+
+	// Create a pattern to search for the font family
+	FcPattern * pattern = FcNameParse(reinterpret_cast<const FcChar8 *>(font_family.c_str()));
+	if (!pattern) {
```

Comment:
check aub altijd expliciet voor `NULL` (bij C API's) of `nullptr` (bij C++ API's)